Makefile

```diff
@@ -96,7 +96,7 @@ acceptance-cluster-setup-kind:
 	@./scripts/acceptance-cluster-setup-kind.sh
 
 test-acceptance: showfocus
-	ginkgo -nodes ${GINKGO_NODES} --slow-spec-threshold ${GINKGO_SLOW_TRESHOLD}s -randomize-all --flake-attempts=${FLAKE_ATTEMPTS} --fail-on-pending acceptance/. acceptance/api/v1/. acceptance/apps/.
+	ginkgo -nodes ${GINKGO_NODES} -slow-spec-threshold ${GINKGO_SLOW_TRESHOLD}s -randomize-all --flake-attempts=${FLAKE_ATTEMPTS} --fail-on-pending acceptance/. acceptance/api/v1/. acceptance/apps/.
```

Comment:
I see `--` in `ginkgo run --help`, accidental change?

Reply:
Not accidental, it simply because the others `test-acceptance-*` that seem to run well on main CI have `-slow-spec-threshold` and not `--slow-spec-threshold` :-). So it seems that the others are faulty in that case ;-) I will fix this.